scram-tools.file/tools/gcc/gcc-cxxcompiler.xml

```diff
@@ -5,6 +5,9 @@
       <environment name="CXX" value="$GCC_CXXCOMPILER_BASE/bin/c++@COMPILER_NAME_SUFFIX@"/>
 %else
       <environment name="CXX" value="c++@COMPILER_NAME_SUFFIX@"/>
+%endif
+if 0%{?rhel} >= 9
+<runtime name="GPROFNG_SYSCONFDIR" value="$GCC_CXXCOMPILER_BASE/etc"/>
```

Comment:
@gartung , please move this outside the `<client></client>` section. I would suggest to add this around line 43 (also properly align it with other `<runtime name` statements)

Reply:
OK